_includes/comparison.html

```diff
@@ -113,6 +123,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
             <td>Traefik Mesh currently does not support transparent TLS encryption.</td>
             <td>Kuma is possibly the most flexible service mesh. Teams should thoroughly consider whether their project can handle the complexity involved.</td>
             <td>OpenServiceMesh (OSM) is the latest service mesh Implementation and simply too young to be production-ready.</td>
+            <td>The different architecture of Cilium limits it's capabilities compared to ther products like TODO.</td>
```

Comment:
Cilium also uses Envoy in addition to eBPF so the capabilities are the same as with other Envoy-based service mesh.